.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Github pages deployment
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Build & Deploy
+        uses: JamesIves/github-pages-deploy-action@master
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          BRANCH: gh-pages # The branch the action should deploy to.
+          FOLDER: dist/ta # The folder the action should deploy.
+          CNAME: ta.choy.in
+          BUILD_SCRIPT: |
+            npm install
+            npm run build
+            cp dist/ta/index.html dist/ta/404.html
+
+on:
```

Comment:
As done in official workflows, put this at the top of the file

Reply:
Done.